mesonbuild/interpreter.py

```diff
@@ -2020,7 +2021,7 @@ def get_cross_property_method(self, args, kwargs):
                     'both_libraries': known_library_kwargs,
                     'library': known_library_kwargs,
                     'subdir': {'if_found'},
-                    'subproject': {'version', 'default_options', 'required'},
+                    'subproject': {'version', 'default_options', 'required', 'method', 'cmake_options'},
```

Comment:
Since the functionality was moved to the module, these should no longer be here, right?

Reply:
Right. Fixed with the last commit.